src/layout/Map/Map.tsx

```diff
@@ -137,6 +148,23 @@ export function Map({
           subdomains={layer.subdomains ? layer.subdomains : []}
         />
       ))}
+      {wmsLayers.map((layer, i) => (
+        <WMSTileLayer
+          key={i}
+          url={layer.url}
+          attribution={layer.attribution}
+          subdomains={layer.subdomains ? layer.subdomains : []}
+          layers={layer.layers}
+          format={layer.format ?? 'image/jpeg'}
+          version={layer.version ?? '1.1.1'}
+          opacity={layer.opacity ?? 1.0}
+          zoomOffset={layer.zoomOffset ?? 0}
+          transparent={layer.transparent ?? false}
+          uppercase={layer.uppercase ?? false}
+          minZoom={layer.minZoom ?? 0}
+          maxZoom={layer.maxZoom ?? 18}
+        />
+      ))}
```

Comment:
When WMS layers are defined in a separate list (after the standard map layers), the layer ordering is a bit restricted. You cannot add a standard layer after a WMS layer as it is now

Reply:
Agreed. Would it be better to combine the WMSLayer and Layer objects in the config, so the ordering can be done there?

Reply:
Yes, I think this can be achieved by using something like:
```ts
new CG.prop(
  'layers',
  new CG.arr(
    new GC.union(
      new CG.obj( /* default layer props */ ),
      new CG.obj( /* wms layer props */ ),
    )
    .setUnionType('discriminated')
  )
  ...
)
...
```

Maybe you also need to add a `type: 'WMS'` property that you can use to check in the `Map` component to know which component to use. For the standard variant a type prop like this needs to be optional (or omitted) for backwards compatibility.